src/components/PostListing.js

```diff
@@ -163,18 +161,19 @@ const PostListing = React.createClass({
   render: function() {
     return (
       <View style={styles.outerListView}>
-        <RefreshableListView
-          ref = {(listView) => {this.listView = listView;}}
-          renderScrollComponent={props => <InfiniteScrollView {...props} />}
+        <ListView
+          ref="listView"
           dataSource={this.state.dataSource}
           renderRow={this.renderRow}
-          loadData={this.props.refresh}
           automaticallyAdjustContentInsets={false}
-          distanceToLoadMore={200}
-          refreshDescription="Refreshing articles"
           style={styles.listView}
-          onLoadMoreAsync={this.props.onLoadMoreAsync}
-          canLoadMore
+          refreshControl={
+            <RefreshControl
+              onRefresh={this.props.refresh}
+              refreshing={this.state.refreshing}
+              title="Refreshing articles" />
+          }
+          onEndReached={this.props.onLoadMoreAsync}
```

Comment:
Think I fixed this, but I ran the app on my device and clicking on an article immediately crashes it. Lets hold off on merging this while I look into it.